app/Http/ValidationRules/Company/ValidExpenseMailbox.php

```diff
@@ -0,0 +1,86 @@
+<?php
+/**
+ * Invoice Ninja (https://invoiceninja.com).
+ *
+ * @link https://github.com/invoiceninja/invoiceninja source repository
+ *
+ * @copyright Copyright (c) 2023. Invoice Ninja LLC (https://invoiceninja.com)
+ *
+ * @license https://www.elastic.co/licensing/elastic-license
+ */
+
+namespace App\Http\ValidationRules\Company;
+
+use App\Libraries\MultiDB;
+use App\Utils\Ninja;
+use Illuminate\Contracts\Validation\Rule;
+
+/**
+ * Class ValidCompanyQuantity.
```

Comment:
i'm not sure we need anything here at all?

Once the user passes the authorization gate, they can configure any domain they wish, the only catch is that it is unique (and only for hosted platform)

Reply:
I would prefer this aporach for 3 reasons:
1. better visibility in frontend, when the ending is already defined/selectable
2. mailservers can be used for multiple domains, maybe it is not wanted to use specific domains
3. endings allows us to also use suffixes on mailboxnames (without any additional implementation)